pylal/coh_PTF_pyutils.py

```diff
@@ -78,19 +78,23 @@ def get_bestnr( trig, q=4.0, n=3.0, null_thresh=(4.25,6), snr_threshold=6.,\
                                     numpy.degrees(trig.dec),\
                                     trig.get_end())
     for ifo in ifos:
-        if ifo.lower()[0] == 'h' :
+        if ifo.lower()[0] == 'h':
             i = ifo.lower()
+        elif ifo.lower()[0] == 'k':
+            i = 't'
         else:
             i = ifo[0].lower()
         sens[ifo] = getattr(trig, 'sigmasq_%s' % i.lower()) * \
                         sum(numpy.array([fPlus[ifo], fCross[ifo]])**2)
     ifos.sort(key=lambda ifo: sens[ifo], reverse=True)
     if len(ifos) > 1:
-        for i in xrange(0, 2):
```

Comment:
You do not want to remove the line `for i in xrange(0, 2):` as the single IFO SNR cut needs to be applied to the two most sensitive detectors (`ifos` is sorted by `sens` above).

The confusing thing here is the double use of `i`.  The loop `for i in xrange(0, 2):` will keep giving you 2 iterations regardless of the `i = ` statements below.  You can try

```
for i in xrange(0,2):
    i = 'a'
    print i
```

to see that.  For clarity, perhaps, change the `i` in `for i in xrange(0, 2):`  into something else.

Reply:
Yes, I see. Then shouldn't it just be...
```
for ifo in ifos[:2]:
```

Reply:
Nice!